services/server/src/sourcify-chains.ts

```diff
@@ -15,6 +15,10 @@ import logger from "./common/logger";
 import fs from "fs";
 import path from "path";
 
+import dotenv from "dotenv";
+
+dotenv.config();
+
```

Comment:
Is this supposed to be included in this PR?

Reply:
Yes this is for https://github.com/ethereum/sourcify/pull/1747/commits/839b3be367f01389f7cb2ab9345908cccfa71f25 i.e. any script using `sourcify-chains.ts` (e.g. `deployContracts.ts`) needs this.